index.html

```diff
@@ -64,7 +73,7 @@ <h2>Software Developer</h2>
                 <li><a href="#"><img src="./assets/icon/twitter.png" alt="Twitter"></a></li>
                 <li><a href="#"><img src="./assets/icon/medium.png" alt="Medium"></a></li>
             </ul>
-                <a class="arrow" href="#"><img src="./assets/icon/element-right.png" alt="arrow"></a>   
+                <a class="arrow" href="#"><img src="./assets/icon/element-right.png" alt="Link image icon for a button"></a>   
```

Comment:
As already mentioned above, this `alt` text would be better implemented as `alt="arrow down"` for example.

Reply:
I have changed the alt text to arrow down text.